lib/system/arc.nim

```diff
@@ -256,6 +256,12 @@ proc isObjWithCache(obj: PNimTypeV2, subclass: cstring, cache: var ObjCheckCache
   else:
     result = isObjSlowPath(obj, subclass, cache)
 
+proc isObjDisplayCheck(source: PNimTypeV2, target: PNimTypeV2, token: uint32): bool {.compilerRtl.} =
```

Comment:
```suggestion
proc isObjDisplayCheck(source: PNimTypeV2, targetDepth: int16, token: uint32): bool {.compilerRtl.} =
```

The compiler can pass `targetDepth` directly here as it knows it. Saves a memory fetch operation.

Reply:
done